### DISCUSSION TITLE
Solve triton problem.

### DISCUSSION THREAD

**Discussion by rbrooks95** [openai/whisper#1375] [Q&A]
This my first time using whisper. When downloading all the dependencies triton (2.0.0) failed. Who do I solve this?

**Comment by darraghenright** [comment]
Were you using Docker? 

If so, maybe this is useful context — I received the same triton (as well as torch) error messages when I tried to install the library on an Alpine container. Based on this [Stack Overflow answer](https://stackoverflow.com/questions/76751155/how-to-install-torch-in-alpine) that's not possible without some work. I ended up using a Debian slim image instead and I had no further problems.

I'm making some assumptions about your scenario, so it might not be relevant, but hope this helps.

**Comment by dgoryeo** [comment]
Came across this therad when I was searching for Triton problems. Was there any resolution to this question? Thanks

**Reply by phineas-pta** [reply]
triton not work on windows

**Reply by dgoryeo** [reply]
Thanks @phineas-pta , I understand triton doesn't work on Windows, but what does that mean to the performance and results of  Whisper? Should we expect to see less functionality, less speed, or less accuracy, and in what degree? Would anyone know?

**Reply by phineas-pta** [reply]
does speed matter in your usecase ?

for e.g. u deploy something and need speed, most likely u use linux anyway

personal use is mostly windows, tbf the time trying compile triton not worth the speed up
